### ISSUE TITLE
Update card headings from H4 to H3

### ISSUE THREAD

**Issue by fionamorrison23** [civictheme/uikit#348]
### Summary

Related to: https://github.com/civictheme/uikit/issues/318

Creating placeholder ticket, details to come. 

### Steps to reproduce

To come

### Observed outcome

To come

### Expected outcome

To come

**Comment by fionamorrison23** [comment]
@alanrako can we discuss? In most cases it would be logical to add an h2 to a list like news, events, publications etc... but in the case of the Navigation card and maybe the Services card, if someone is using those as navigation then it's likely they wouldn't put an h2 on the page above the cards.
FYI @richardgaunt

**Comment by alanrako** [comment]
@fionamorrison23 
Yes lets discuss. Please add this to the backlog for discussion.
